schema.gql

```diff
@@ -69,6 +107,14 @@ type SpotType {
   personaUUIDs: [String!]
 }
 
+input UpdatePersonaInput {
+  name: String
```

Comment:
![image](https://user-images.githubusercontent.com/7398214/72114545-38164e00-3344-11ea-9f2e-2f90da46c575.png)
Maybe it would be good for a Persona to consist of Card and Wall/Page (you can choose). Because card and wall can have image and avatar.

Reply:
Ok so lets continue this conversation on Slack